src/transit.js

```diff
@@ -337,17 +339,44 @@ class Transit {
 			return;
 		}
 
-		// Remove pending request
-		this.removePendingRequest(id);
-
 		this.logger.debug(`Response '${req.action.name}' received from '${packet.sender}'.`);
 
 		// Update nodeID in context (if it uses external balancer)
 		req.ctx.nodeID = packet.sender;
 
 		// Merge response meta with original meta
 		_.assign(req.ctx.meta, packet.meta);
-
+		//Stream case
+		//get the underlined stream for id
+		if(packet.stream !== undefined && packet.stream !== "undefined"){
+			let pass = this.pendingStreams.get(id);
+			if(!packet.stream && pass){
+				//end of  stream
+				pass.end();
+				// Remove pending request
+				this.removePendingRequest(id);
+				this.pendingStreams.delete(id);
+			}
+			if(packet.stream && pass){
+				//on stream chunk
+				pass.write(packet.data.type === "Buffer" ? new Buffer.from(packet.data.data):packet.data);
+			}
+			else{
+				//create a new pass stream
+				pass = new Transform({
+					transform:function(chunck,encoding,done){
```

Comment:
Please fix typo from `chunck` to `chunk`

Reply:
sure !